cibuild.cmd

```diff
@@ -137,6 +138,7 @@ exit /b 0
 :BuildFailed
 echo Build failed with ERRORLEVEL %ERRORLEVEL%
 call :TerminateBuildProcesses
+:ActuallyFail
```

Comment:
Please use a more descriptive name here.  For example: TerminateBuildProcessesFailed.